onnxruntime/core/providers/webgpu/webgpu_utils.h

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <cstdint>
+
+namespace onnxruntime {
+namespace webgpu {
+
+int64_t GetMaxComponents(int64_t size);
```

Comment:
considering the function body is super simple, I suggest to include the function body here so that actually it's self-explanatory and easier to understand. don't forget to add `inline` to the function.